includes/admin/upgrades/migrations/migration-18_5.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * @package WPBDP\Admin\Upgrades\Migrations
+ */
+
+/**
+ * Migration for DB version 18.5
+ */
+class WPBDP__Migrations__18_5 extends WPBDP__Migration {
+
+	/**
+	 * If payments were on, we disable all free plans and vice versa.
+	 * We scan all enabled plans first for this migration.
+	 * If payments was enabled, we disable the free plans.
+	 * If payments was disabled, we disabled the paid plans.
+	 *
+	 * @since x.x
+	 */
+	public function migrate() {
+		global $wpdb;
+		$payments_on = wpbdp_get_option( 'payments-on' );
+		$sql         = "SELECT `id`, `amount` FROM {$wpdb->prefix}wpbdp_plans WHERE `enabled` = %d";
```

Comment:
I think we need to reverse this to enabled != 0. Plans may get missed this way.

Also, let's be consistent and not use the ``